simple_graph/test_sgraph.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+import sgraph
+
+
+def test___init__():
+    """Test instantiates simple graph with dictionary."""
+    assert sgraph.Sgraph().dict == {}
+
+
+def test_nodes(empty_graph, populated_graph):
+    """Test return list of all nodes in a graph"""
+    assert empty_graph.nodes() == []
```

Comment:
This should be 2 different tests. You're testing 2 distinct things, and if this test fails you don't know which is broken.